### PR TITLE
docs: translating to Spanish complex-python-package-builds.md

### PR DESCRIPTION
**Pull request by RobPasMue**
Related to #313

### CODE REVIEW

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 1]
```diff
@@ -648,13 +648,15 @@ msgstr ""
...
 msgid "Complex Python package builds"
-msgstr ""
+msgstr "Compilaciones complejas de paquetes de Python"
 
 #: ../../package-structure-code/complex-python-package-builds.md:3
 msgid ""
 "This guide is focused on packages that are either pure-python or that "
 "have a few simple extensions in another language such as C or C++."
 msgstr ""
+"Esta guía se centra en paquetes que son Python puros o que tienen unas "
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:658]:
```suggestion
"Esta guía se centra en paquetes que solamente utilizan Python o que tienen unas "
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 2]
```diff
@@ -669,17 +671,31 @@ msgid ""
 "an overview and thorough discussion of these nuances, please see [this "
 "site.](https://pypackaging-native.github.io/)"
 msgstr ""
+"En el futuro, queremos proporcionar recursos para flujos de trabajo de "
+"empaquetado que requieran compilaciones más complejas. Si tiene preguntas "
+"sobre estos tipos de paquetes, por favor [agregue una pregunta a nuestro "
+"foro](https://pyopensci.discourse.group/) o abra una [solicitud sobre esta guía "
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:677]:
```suggestion
"foro](https://pyopensci.discourse.group/) o abra una [issue sobre esta guía "
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 3]
```diff
@@ -669,17 +671,31 @@ msgid ""
...
+"específicamente en el repositorio](https://github.com/pyOpenSci/python-package-guide/issues). "
+"Hay muchos matices para construir y distribuir paquetes de Python que tienen "
+"extensiones compiladas que requieren dependencias que no están desarrolladas en Python en tiempo de "
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:680]:
```suggestion
"extensiones compiladas que requieren dependencias en otros lenguajes en el momento de "
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 4]
```diff
@@ -669,17 +671,31 @@ msgid ""
...
+"compilación. Para obtener una descripción general y una discusión exhaustiva "
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:681]:
```suggestion
"construcción del paquete. Para obtener una descripción general y una discusión exhaustiva "
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 5]
```diff
@@ -669,17 +671,31 @@ msgid ""
...
+msgstr "Paquetes de Python puros vs. paquetes con extensiones en otros lenguajes"
 
 #: ../../package-structure-code/complex-python-package-builds.md:10
 msgid ""
 "You can classify Python package complexity into three general categories."
 " These categories can in turn help you select the correct package "
 "frontend and backend tools."
 msgstr ""
+"Se puede clasificar la complejidad de los paquetes de Python en tres categorías. "
+"Estas categorías, a su vez, pueden ayudarlo a seleccionar las herramientas de "
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:697]:
```suggestion
"Estas categorías, a su vez, pueden ayudarlo a seleccionar las mejores herramientas de "
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 6]
```diff
@@ -669,17 +671,31 @@ msgid ""
...
+"frontend y backend correctas."
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:698]:
```suggestion
"frontend y backend."
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 7]
```diff
@@ -688,6 +704,10 @@ msgid ""
 "chose a tool below that has the features that you want and be done with "
 "your decision!"
 msgstr ""
+"**Paquetes de Python puros:** son paquetes que solo dependen de Python para "
+"trabajar. Construir un paquete de Python puro es más simple. Como tal, puede "
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:708]:
```suggestion
"funcionar. Construir un paquete de Python puro es más simple. Como tal, puede "
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 8]
```diff
@@ -688,6 +704,10 @@ msgid ""
...
+"elegir una herramienta a continuación que tenga las características que desea y, "
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:709]:
```suggestion
"elegir una herramienta a continuación que tenga las características que desea "
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 9]
```diff
@@ -688,6 +704,10 @@ msgid ""
...
+"¡tomar una decisión!."
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:710]:
```suggestion
"sin tener que considerar muchos detalles."
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 10]
```diff
@@ -700,22 +720,38 @@ msgid ""
 "that supports additional build setups. We suggest that you chose build "
 "tool that supports custom build steps like Hatch."
 msgstr ""
+"**Paquetes de Python con extensiones en otros lenguajes:** Estos paquetes tienen "
+"componentes adicionales llamados extensiones escritas en otros lenguajes (como C o C++). "
+"Si tiene un paquete con extensiones no escritas en Python, entonces debe seleccionar "
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:725]:
```suggestion
"Si tiene un paquete con extensiones no escritas en otros lenguajes, debe seleccionar "
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 11]
```diff
@@ -700,22 +720,38 @@ msgid ""
...
+"una herramienta de backend de compilación que permita pasos de compilación adicionales "
+"necesarios para compilar su código de extensión. Además, si desea utilizar una herramienta "
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:727]:
```suggestion
"necesarios para compilar su código en estos lenguajes. Además, si desea utilizar una herramienta "
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 12]
```diff
@@ -700,22 +720,38 @@ msgid ""
...
+"de frontend para ayudar a su flujo de trabajo, deberá seleccionar una herramienta que admita "
+"configuraciones de compilación adicionales. Sugerimos que elija una herramienta de compilación "
+"que admita pasos de compilación personalizados como Hatch."
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:730]:
```suggestion
"que admita pasos de compilación personalizados como por ejemplo Hatch."
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 13]
```diff
@@ -742,6 +785,14 @@ msgid ""
 "[plugins](https://hatch.pypa.io/1.9/plugins/about/) or be replaced by a "
 "backend that is already capable of building extension modules."
 msgstr ""
+"En esta guía de empaquetado, recomendamos usar `hatch` junto con su backend "
+"preferido `hatchling`. Si bien esto será adecuado para la mayoría de los paquetes, "
+"se puede usar un backend alternativo con Hatch si es necesario al crear un módulo de "
+"extensión. Un módulo de extensión de Python es uno que está compuesto, ya sea en parte "
+"o en su totalidad, de código compilado. En este caso, el backend elegido (como `meson-python`) "
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:792]:
```suggestion
"o en su totalidad, de código compilado en otro lenguaje. En este caso, el backend elegido (como `meson-python`) "
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 14]
```diff
@@ -742,6 +785,14 @@ msgid ""
...
+"debe saber cómo compilar el lenguaje de extensión y vincularlo a Python. `hatchling` no sabe "
+"cómo hacer esto por sí solo y debe hacer uso de [plugins](https://hatch.pypa.io/1.9/plugins/about/) "
+"o ser reemplazado por un backend que ya sea capaz de construir módulos de extensión."
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:795]:
```suggestion
"o cambiar un backend que sea capaz de construir módulos de extensión."
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 15]
```diff
@@ -750,6 +801,9 @@ msgid ""
 " command, or from following the packaging tutorial, you may have to make "
 "a change like this"
 msgstr ""
+"Para usar un backend diferente, deberá editar el `pyproject.toml` de su proyecto. "
+"Si tiene un `pyproject.toml` generado por el comando `hatch`, o siguiendo el tutorial "
+"de empaquetado, es posible que deba hacer un cambio como este"
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:806]:
```suggestion
"de empaquetado, probablemente tendrá hacer un cambio como este"
```

**Review by flpm — COMMENTED** [review]
Just some minor changes

**Review by RobPasMue — COMMENTED** [review]
Thanks a lot @flpm!

**Review by flpm — APPROVED** [review]
LGTM

**Pull request merged**